openml/flows/flow.py

```diff
@@ -322,7 +322,16 @@ def _from_dict(cls, xml_dict):
         arguments['tags'] = tags
 
         arguments['model'] = None
-        return cls(**arguments)
+        flow = cls(**arguments)
+
+        if 'sklearn' in arguments['external_version']:
```

Comment:
I will replace this with `startswith('sklearn')`. In the long run, we probably need to build a plugin-like system in which the converters can register themselves for an 'external_version' string.